_posts/en/newsletters/2022-10-05-newsletter.md

```diff
@@ -0,0 +1,188 @@
+---
+title: 'Bitcoin Optech Newsletter #220'
+permalink: /en/newsletters/2022/10/05/
+name: 2022-10-05-newsletter
+slug: 2022-10-05-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- **Proposed new transaction relay policies designed for LN-penalty:**
+  Gloria Zhao [posted][] to the Bitcoin-Dev mailing list a proposal to
+  allow transactions to opt-in to a modified set of transaction relay
+  policies.  Any transaction which sets its version parameter to `3`
+  will:
+
+    - Be replacable while it is unconfirmed by a transaction paying a
+      higher feerate and a higher total fee (the current main
+      [RBF][topic rbf] rules)
+
+    - Be required to set its version parameter to `3` and follow all of
+      these rules if any of the transactions it depends on (its ancestor
+      transactions) are both unconfirmed and also version `3`
+
+    - Be rejected if any of its unconfirmed ancestors already have any
```

Comment:
```suggestion
    - Be rejected if any of its unconfirmed V3 ancestors already have any
```

I know it's draft, but wanted to correct this. Non-V3 transactions' descendant rules are still mostly respected. I do apologize as I know the rules are pretty convoluted/hacky.

To illustrate what I mean (where <- means "spends," i.e. parent <- child. "large" means >1000vB):
![image](https://user-images.githubusercontent.com/25183001/193531846-01e46645-62b7-4e07-82a6-9c61e87387a1.png)